src/Magnum/Shaders/Generic.h

```diff
@@ -478,6 +492,11 @@ struct BaseGeneric {
     #ifndef MAGNUM_TARGET_GLES2
     typedef GL::Attribute<4, UnsignedInt> ObjectId;
     #endif
+    typedef GL::Attribute<6, Vector4> Weights;
+    typedef GL::Attribute<7, Vector4> JointIds;
+
+    typedef GL::Attribute<10, Vector4> SecondaryWeights;
+    typedef GL::Attribute<11, Vector4> SecondaryJointIds;
```

Comment:
Seeing #444, this PR should contain also the `generic.glsl` changes and corresponding additions to `GenericTest::glslMatch()`